### PR TITLE
Fixes for vscode 1.75.0

### PR DESCRIPTION
**Pull request by pokey**
## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [ ] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [ ] I have not broken the cheatsheet

### CODE REVIEW

File: docs/contributing/CONTRIBUTING.md [hunk 1]
```diff
@@ -18,15 +18,7 @@ extension](#running--testing-extension-locally), you may want to check out the
 
 1. Clone [`cursorless`](https://github.com/cursorless-dev/cursorless)
 2. Open the newly created `cursorless` directory in VSCode. If you're on Windows, don't use WSL (see [#919](https://github.com/cursorless-dev/cursorless/issues/919) for discussion / workaround).
-3. Add the following to your [VSCode `settings.json`](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) (can be opened by saying `"show settings json"` with a recent install of knausj)
```
Comment (pokey) [inline docs/contributing/CONTRIBUTING.md:21]: no longer necessary as profiles are now generally available

File: src/apps/cursorless-vscode-e2e/suite/editNewCell.test.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ async function runTest(
 
   // FIXME: There seems to be some timing issue when you create a notebook
   // editor
-  await sleepWithBackoff(1000);
+  await sleepWithBackoff(100);
```
Comment (pokey) [inline src/apps/cursorless-vscode-e2e/suite/editNewCell.test.ts:33]: too much sleep; makes things slow when combined with backoff

File: src/apps/cursorless-vscode-e2e/suite/editNewCell.test.ts [hunk 3]
```diff
@@ -50,6 +50,8 @@ async function runTest(
     ],
   });
 
+  await sleepWithBackoff(100);
```
Comment (pokey) [inline src/apps/cursorless-vscode-e2e/suite/editNewCell.test.ts:53]: looks like vscode 1.75.0 breaks a new promise 😅

**Review by AndreasArvidsson — DISMISSED** [review]

**Review by AndreasArvidsson — DISMISSED** [review]

**Comment by pokey** [comment]
To try:

- [ ] Update vscode test runner library
- [ ] Disable all / subset of test (eg notebook tests)

**Comment by pokey** [comment]
(Windows failures tracked in https://github.com/microsoft/vscode/issues/173681)

**Review by AndreasArvidsson — APPROVED** [review]

**Pull request merged**